### DISCUSSION TITLE
Crypto trading

### DISCUSSION THREAD

**Discussion by lavoiekeven** [ahmedsakr/wstrade-api#69] [Q&A] [answered]
Any plan to support crypto buying and selling?

I just updated from 0.11 to the lastest version and it is a nice improvement!
Thanks.

**Comment by ahmedsakr** [comment]
I am happy you're liking the new version of `wstrade-api`.

Crypto support has been on my mind and it'll become a priority now. I'll raise an issue for it. I probed the necessary API implementations and it doesn't seem to be a significant rework to smoothly support crypto trading.

**Reply by lavoiekeven** [reply]
Super thanks!

**Comment by bwilliamson718** [comment]
would like to see this more active.

**Reply by ahmedsakr** [reply]
I'll try my best to have it supported in the next week or so.

**Comment by ahmedsakr** [comment, marked as answer]
@lavoiekeven  @shokitaka  Crypto trading is now supported in `v1.3.0`

No major API changes. All you have to do is specify `CC` (cryptocurrency) as the exchange for your crypto trades, and it should work!


```javascript
(async () => {
    // assuming login stuff already done...
    let accs = await trade.accounts.all();
    console.log(await trade.orders.marketBuy(accs.crypto, 'BTC:CC', 0.0033));
})().catch(error => console.log(error)); 
```
